src/vs/workbench/contrib/chat/browser/actions/chatCodeblockActions.ts

```diff
@@ -88,8 +88,7 @@ export function registerChatCodeBlockActions() {
 				icon: Codicon.copy,
```

Comment:
this is just reverting changes I had made to get our actions to appear before the ones in the chat toolbar